inclusion_connect/templates/account_activation.html

```diff
@@ -0,0 +1,40 @@
+{% extends "layout/base.html" %}
+{% load bootstrap4 %}
+{% load inclusionconnect_fields %}
+
+{% block content %}
+
+    <h1 class="fr-h4">Renseignez votre mot de passe</h1>
+
+    <div>
+        Vous pouvez réutiliser celui de votre compte sur {{ application_name }} <strong>s'il répond aux critères de sécurité</strong>.
+    </div>
+
+    <hr>
+
+    {{ last_name }} {{ first_name }} ({{ email }})
+
+    <form method="post" action="" class="js-prevent-multiple-submit">
```

Comment:
c.f. les correctifs HTML de Xavier, et https://html.spec.whatwg.org/#attr-fs-action.
```suggestion
    <form method="post" class="js-prevent-multiple-submit">
```

Reply:
bien noté, je vais le retirer de partout